### ISSUE TITLE
Retrieving data from fluent (MySQL) in a command is empty most of the time

### ISSUE THREAD

**Issue by Casperhr** [vapor/vapor#565]
The command
Register it in main.swift
```
import Console
import Fluent

public final class NewsSyncer: Command {
    public let id = "sync:news"
    
    public let help: [String] = [
        "Sync news"
    ]
    
    public let console: ConsoleProtocol
    
    public init(console: ConsoleProtocol) {
        self.console = console
    }
    
    public func run(arguments: [String]) throws {
        
        do {
            let races = try Race.all()
            print(races)
        } catch {
            print(error)
        }
    }
}
```

Terminal log
```
-> vapor run sync:news
Running ironman...
[]
casper_r @ Caspers-MacBook-Pro-2.local [ ~/vapor/ironman ]
-> vapor run sync:news
Running ironman...
[]
casper_r @ Caspers-MacBook-Pro-2.local [ ~/vapor/ironman ]
-> vapor run sync:news
Running ironman...
[]
casper_r @ Caspers-MacBook-Pro-2.local [ ~/vapor/ironman ]
-> vapor run sync:news
Running ironman...
[]
casper_r @ Caspers-MacBook-Pro-2.local [ ~/vapor/ironman ]
-> vapor run sync:news
Running ironman...
[App.Race, App.Race, App.Race]
```

It seems random, maybe it's something today with delay time from run to command is actually running

**Comment by tannernelson** [comment]
I'm guessing an error is being thrown in the init for one of the models but we're not seeing it since you're still on 0.16. Once we finally update the MySQL drivers we will get better error messages here.

**Comment by Casperhr** [comment]
Ok, let me know when I can test that again

**Comment by Casperhr** [comment]
I discovered a "funny" thing. vapor xcode --mysql fixes this issue. It's new models you have the problem on. Can also confirm it happened with a raw sql. Also through the MySQL.Database object

**Comment by Casperhr** [comment]
This seems to be fixed in 1.0
